public/basehandle.h

```diff
@@ -13,26 +13,11 @@
 
 #include "const.h"
 #include "tier0/dbg.h"
+#include "ihandleentity.h"
 
+typedef CEntityHandle CBaseHandle;
 
-class IHandleEntity;
-
-// Represents EHANDLE2 class
-// TODO: GAMMACASE: Replace old CBaseHandle with this
-class CEntityHandle
-{
-public:
-	union
-	{
-		uint32 m_Index;
-		struct
-		{
-			uint32 m_EntityIndex : 15;
-			uint32 m_Serial : 17;
-		} m_Parts;
-	};
-};
-
+#if 0

```

Comment:
This file is better to be removed then if you are moving to use entityhandle.h. And all the includes corrected